### ISSUE TITLE
.hbs highlight is not working

### ISSUE THREAD

**Issue by mthnglac** [alexlafroscia/tree-sitter-glimmer#44]
nvim-treesitter does not highlight the .hbs template. To be sure of the problem, I deleted all my config, installed nvim-treesitter & glimmer parser and then applied highlight setting with nvim-treesitter. However, nothing changed.

**Comment by alexlafroscia** [comment]
Can you show me what your configuration looks like? I need some kind of reproduction steps to try to help, as it's working alright for me.

**Comment by mthnglac** [comment]
When I did "set ft=handlebars", I expected treesitter to recognize the code and color it, but it didn't. For now, I have installed the "vim-mustache-handlebars" plugin as a temporary solution. This plugin does its own coloring after "set ft=mustache" setting.

**Comment by mthnglac** [comment]
By the way my telescope.nvim plugin works with nvim-treesitter and somehow it highlights the handlebar template correctly.

**Comment by rwjblue** [comment]
FWIW, I submitted https://github.com/nvim-treesitter/nvim-treesitter/pull/1973 which I think should fix this (at least in my use case).

tldr; **someone** has to setup the `*.hbs` association with `handlebars`.

**Comment by alexlafroscia** [comment]
Thanks for that @rwjblue!

@mthnglac it seems odd that `ft=handlebars` _does not_ work in a buffer, but your Telescope setup _does_ have it work.... it'll be interesting to see if that landing will fix the problem that @mthnglac is having!

**Comment by rwjblue** [comment]
Good point!!! I submitted https://github.com/nvim-treesitter/nvim-treesitter/pull/1974 to ensure that our parser is setup for `html.handlebars`.

**Comment by mthnglac** [comment]
https://github.com/nvim-treesitter/nvim-treesitter/pull/1973

I just got treesitter's update and I think this PR solved my problem. Now I can get a healthy highlight like in the telescope plugin window. `:set ft` command output is `html.handlebars` and highlight is working. Thanks for the support and answers.

**Comment by rwjblue** [comment]
FWIW @mthnglac you will only get partial highlighting support (until https://github.com/nvim-treesitter/nvim-treesitter/pull/1974 lands) if your filetype is `html.handlebars`. It'll highlight like `html` and not have special highlighting for the handlebars / mustache portions.

**Comment by mthnglac** [comment]
after the last PR ([#1973](https://github.com/nvim-treesitter/nvim-treesitter/pull/1973)) that I mentioned, my `filetype` output is `handlebars`. Thus, my problem was fixed I guess.
